### ISSUE TITLE
Add Parameter does not work

### ISSUE THREAD

**Issue by BornSupercharged** [yblis/OllamaManagerUI#2]
window.addParameter fails due to Element parametersList not found

I did a grep and did not find any element named parametersList in the repo

**Comment by yblis** [comment]
When did you encounter an error?

I just added a system prompt parameterization feature.

**Comment by BornSupercharged** [comment]
I tested it about 5 hours ago, I see you made changes to it and not it doesn't have that error anymore. 

I've just updated my PR to include the text translations to account for your changes.

**Comment by yblis** [comment]
I saw the translations, I was also working on them.

**Comment by BornSupercharged** [comment]
Are you using an AI agent to help you write this? Just noticing Agent Queries in the recent commits and curious how you're going about doing that if so

**Comment by yblis** [comment]
Yes, I use an AI agent, even though it makes a lot of mistakes so I have to correct quite a few things, but it saves me time. And it's the agent that makes the commits.
